DESCRIPTION

```diff
@@ -53,9 +54,10 @@ Imports:
     viridisLite
 Suggests:
     aws.s3,
-    ggmap (>= 3.0.0),
     knitr,
     rmarkdown,
+    rosm,
+    prettymapr,
```

Comment:
where is prettymapr used? ggspatial imports rosm, so I believe that means they should be either both depends, or both suggests.

I think all should be suggests, and bioRad should be able to run without it. When map is run, we should do a `rlang::check_installed()` test (see example in previous basemap.R) that verifies if ggspatial is installed.